docs/topics/items.rst

```diff
@@ -401,7 +401,7 @@ In code that receives an item, such as methods of :ref:`item pipelines
 :func:`~itemadapter.is_item` function to write code that works for
 any :ref:`supported item type <item-types>`:
 
-.. autoclass:: itemadapter.ItemAdapter
+The itemadapter.ItemAdapter class is a wrapper for data container objects, providing a common interface to handle objects of different types in an uniform manner, regardless of their underlying implementation. see the: `itemadapter.ItemAdapter Documentation <https://github.com/scrapy/itemadapter>`_.
 
 .. autofunction:: itemadapter.is_item
```

Comment:
This needs to be removed too.

And I think it's fine to just remove both and replace the semicolon with a period in the preceding statement, as both `itemadapter.ItemAdapter` and `itemadapter.is_item` are already linked in that statement.